src/server/auth/db.go

```diff
@@ -0,0 +1,20 @@
+package auth
+
+import (
+	"github.com/jmoiron/sqlx"
+	"golang.org/x/net/context"
+)
+
+// TODO(acohen4): Decide on hash as primary key or unique column
+// CreateTokensTable sets up the postgres table which tracks active clusters
+func CreateAuthTokensTable(ctx context.Context, tx *sqlx.Tx) error {
+	_, err := tx.ExecContext(ctx, `
+CREATE TABLE IF NOT EXISTS auth.auth_tokens (
+	token_hash VARCHAR(4096) PRIMARY KEY,
+	subject VARCHAR(64) NOT NULL,
```

Comment:
We're probably going to want an index on the subject and expiration as well? I think subject could just be a normal b-tree index, we might benefit from a different type for expiration to do efficient deletions when we clean them up?

Reply:
Oh no, I meant a different type of index, but it seems like a b-tree might actually be fine. Don't let me prematurely optimize this too hard ;)

Reply:
On second thought, do we have an anticipated use-case in mind to query on subject? My only concern is adding unused indexes that may slow down writes. 

Although this is such a simple table it may not matter too much. What do you think @actgardner ?

Reply:
I have a follow-on task to add an RPC for "revoke all tokens for a given user", I think indexing on subject is useful in that case.

Reply:
sounds good :)